ament_tools/build_types/ament_python.py

```diff
@@ -318,6 +318,9 @@ def _install_action_python(self, context):
         items = ['setup.py']
         # add all first level packages
         items += [p for p in context['setup.py']['packages'] if '.' not in p]
+        # relative python-ish paths are allowed as entries in py_modules, see:
+        # https://docs.python.org/3.5/distutils/setupscript.html#listing-individual-modules
+        items += [re.sub(r'\.', os.path.sep, p) for p in context['setup.py']['py_modules']]
```

Comment:
That's true, I was thinking it would be a more complicated substitution but ended up simplifying it. I'll open a pr to change it.